src/components/StatVideoCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+
+export default function StatVideoCard({video}) {
+    const [url, setUrl] = useState(video.url)
+  return (
+    <div className='w-full h-24 bg-[#464545] flex justify-between items-center text-white p-2'>
+        <div className=''>
+            <div className='text-5xl font-bold'>{video.title}</div>
+            <div>
+            <div className='text-sm'>https://www.youtube.com/</div>
+            </div>
+        </div>
+            <div className='h-full flex flex-col items-end'>
+                <div className='text-sm font-bold h-1/4'>LENGTH: XX:XX</div>
```

Comment:
Use the length passed in from prop here. Don't worry about formatting it to look like time with colons.